string picture_file_name
---
bool success
